src/usePopperMarginModifiers.tsx

```diff
@@ -61,5 +61,40 @@ export default function usePopperMarginModifiers(): [
       },
     };
   }, [margins]);
-  return [callback, [modifier]];
+
+  // Converts popover arrow margin to arrow modifier padding
+  const popoverArrowMargins = useMemo(() => {
+    return {
+      name: 'popoverArrowMargins',
+      enabled: true,
+      phase: 'main',
+      requiresIfExists: ['arrow'],
+      effect({ state }) {
+        if (
+          !overlayRef.current ||
+          !state.elements.arrow ||
+          !hasClass(overlayRef.current, 'popover') ||
+          !state.modifiersData['arrow#persistent']
```

Comment:
**question:** where does `#persistent` come from? is this really an `id`?

Reply:
its a special key Popper adds, nothing unique about it tho aside from Popper picked it